backend/apps/permissions/migrations/0034_auto_20221114_1854.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.16 on 2022-11-14 17:54
+
+from django.db import migrations
+from apps.permissions.constants import ADMIN_GROUP_TYPE, MEMBER_GROUP_TYPE
+
+
+def migrate_org_groups(apps, schema_editor):
+    ResponsibleGroup = apps.get_model("permissions", "ResponsibleGroup")
+    for responsible_group in ResponsibleGroup.objects.all():
+        if responsible_group.name == "HR":
+            # ResponsibleGroup names format is "Organization:Type" (e.g. "Organization:ADMIN")
+            # Type is either "ADMIN" or "MEMBER"
+            # ResponsibleGroup has a group field which is a django.contrib.auth.models.Group with outdated names
+            # Change the Type in group name to "ADMIN" or "MEMBER" depending on the ResponsibleGroup name (e.g. "Organization:HR:uuid" -> "Organization:ADMIN:uuid")
+            # Change ResponsibleGroup.group_type from "HR" or "PRIMARY" to "ADMIN" or "MEMBER"
+            responsible_group.name = f"{responsible_group.organization.name}:{ADMIN_GROUP_TYPE}"
+            responsible_group.group.name = (
+                f"{responsible_group.organization.name}:{ADMIN_GROUP_TYPE}:git{responsible_group.group.uuid}"
```

Comment:
This line fails as `group` has no attribute `uuid`
```suggestion
                f"{responsible_group.organization.name}:{ADMIN_GROUP_TYPE}:git{responsible_group.group.uuid}"
```